code/iaas/logic/src/main/java/io/cattle/platform/process/instance/InstanceStart.java

```diff
@@ -167,7 +171,9 @@ protected void waitForDependenciesStart(Instance instance) {
                     new ResourcePredicate<Instance>() {
                         @Override
                         public boolean evaluate(Instance obj) {
-                            return validateState(obj);
+                            InstanceHostMap ihm = objectManager.findAny(InstanceHostMap.class, INSTANCE_HOST_MAP.INSTANCE_ID, obj.getId(), 
```

Comment:
I don't think is taking into account "startOnce" containers as it did before.  In this situation the ihm maybe be "inactive" yet valid.